snap/snapcraft.yaml

```diff
@@ -18,6 +18,14 @@ apps:
       - network-observe
       - network
       - home
+   discover:
```

Comment:
and also, I should we should just name this command `discover-tempest-config` to avoid adding one more confusion to the already confusing, python-tempestconf, discover-tempest-config situation, how does that sound?

Reply:
I'm using aliases, so `discover-tempest-config` should be there. Naming this `discover-tempest-config` it will mean that you need to run `tempest.discover-tempest-config` and that does not look good to me. Like this you will run `tempest.discover` or `discover-tempest-config`.

Reply:
Since these are upstream projects, I think we need to be careful when we are introducing new stuff. I don't see the benefit of having `tempest.discover` over `tempest.discover-tempest-config` really. I would like to mention I am against this change. However, it is up to the maintainers to decide what goes in or not.